documentation/docs/install-pmm/install-pmm-server/baremetal/docker/remove_container.md

```diff
@@ -4,8 +4,9 @@
 
     !!! summary alert alert-info ""
```

Comment:
Should we delete this summary?
Since the page already contains only this task, the summary seems to duplicate the existing content. Summary sections are more appropriate for long topics where they help users jump to specific sections.